Migration script/openIMIS migration v1.4.2 - v1.4.3.sql

```diff
@@ -0,0 +1,1097 @@
+--- MIGRATION Script from v1.4.2 to v1.4.3
+
+-- OTC-67: RFC-116 New state of policies
+
+IF COL_LENGTH('tblIMISDefaults', 'ActivationOption') IS NULL
+BEGIN
+	ALTER TABLE tblIMISDefaults ADD ActivationOption tinyint NOT NULL DEFAULT 2
+END
+
+-- ************************************************************************************************************
+-- Migration of earlier changes
```

Comment:
The UUID part is valid only for TZ and Cameroon. Remove this from here and create a separate file on your computer which you run when you upgrade the servers.

Reply:
Part of UUID has been removed and added to the new migration saved on the computer. The new migration will be invoked directly in the database on the server